snap/snapcraft.yaml

```diff
@@ -54,6 +54,29 @@ parts:
     stage:
       - snapcraft-completion
 
+  snapcraft-scripts:
+    source: snap/local
+    plugin: dump
+    organize:
+      craft.git: libexec/snapcraft/craft.git
+    stage:
+      - "libexec/"
+
+  git:
+    plugin: nil
+    stage-packages: [git]
+    build-attributes:
+      - enable-patchelf
+    prime:
+      - "-usr/bin"
+      - "-usr/share/doc"
+      - "-usr/share/man"
+      # perl is part of the core22 / core24
+      - "-usr/share/perl"
+      - "-usr/lib/x86_64-linux-gnu/perl"
+      - "-usr/lib/x86_64-linux-gnu/libperl*"
+      - "-usr/lib/x86_64-linux-gnu/libgdbm*"
+
```

Comment:
It might be more readable to invert the logic and list the files/dirs to keep, rather than those to omit.  Looks like from the git `.deb` you intend to keep `/usr/lib/git-core` and some bash completion stuff.

Reply:
Good idea!
I haven't paid attention to that, as we wanted to replace it with chiselled version at some point